src/main/AndroidManifest.xml

```diff
@@ -8,9 +8,13 @@
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
 
+    <uses-permission android:name="android.permission.PACKAGE_USAGE_STATS"/>
+    <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
RADAR-Android-Commons already states `INTERNET` permission, no need to repeat it. `INTERNET` permission is not used here, is it?